lib/navigation/app_controller.dart

```diff
@@ -56,6 +61,12 @@ class _AppControllerState extends State<AppController> {
           unSelectedColor: Colors.grey,
           backgroundColor: Colors.white,
         )
+      ),
+      // Temporary until profile and chat views are modified
+      // to look like the new mockups
+      theme: ThemeData(	
+        primaryColor: Color(0xFF0F1236),	
+        accentColor: Color(0xFF45cab9),	
```

Comment:
@HelenGezahegn I added this back just because a lot of files are referencing these colours at the moment, but we can defs delete them again once we're done changing our views for the new mockups!